draft/draft-ietf-ippm-twamp-yang.xml

```diff
@@ -485,50 +491,53 @@ o-------------------o                       o-------------------o
         specifies the Mode values according to their preferred order of use by
         the operator of this Control-Client, including the authentication and
         encryption Modes. Specifically, mode-preference-chain lists the mode
-        and its corresponding priority, expressed as a 16-bit unsigned
-        integer, where zero is the highest priority and subsequent integers
-        increase by one.</t>
+        and its corresponding priority, as a 16-bit unsigned integer. Values
+        for the priority start with zero, the highest priority, and decreasing
+        priority value is indicated by every increase in value by one.</t>
 
         <t>Depending on the Modes available in the Server Greeting, the
         Control-Client MUST choose the highest priority Mode from the
         configured mode-preference-chain list.</t>
 
-        <t>Note that the list of preferred Modes may set bit position
-        combinations when necessary, such as when referring to the extended
-        TWAMP features in <xref target="RFC5618">Mixed Security Mode for TWAMP
+        <t>Note that the list of preferred Modes may set bit multiple bit
+        position independently, such as when referring to the extended TWAMP
```

Comment:
s/position/positions/

Reply:
There's also an unneeded 'bit' before multiple in "may set bit multiple bit"?

Reply:
Will fix both.